default.nix

```diff
@@ -35,14 +35,12 @@ in makerScriptPackage {
     dss-deploy
   ];
 
-  # Patch scripts by removing `cd` commands and adding DAPP_OUT env so that
-  # `dapp create` can find the right binaries even though no submodules have
-  # been downloaded inside the Nix build environment.
+  # Patch scripts by removing `cd` commands and `./bin/` from `dss-deploy`
+  # script path.
   patchBin = writeScript "remove-cd" ''
     #!${stdenv.shell}
     exec ${perl}/bin/perl -pe '
-      s|^(\s*)cd\s+(?:.*/)*([^\s/\n\r\s;&\|\.]+)|\1export DAPP_OUT=\$DAPP_LIB/\2/out|;
-      s|^(\s*)cd\s+[^\n\r;&\|]+|\1|;
+      s|^(\s*)cd\s+[^\n\r;&\|]+|\1true|;
```

Comment:
How come we're adding `true` here?

Reply:
To avoid `cd ./mjau/mjau || exit 1` becoming ` || exit 1` but instead `true || exit 1`.

But this will be moot if we complete #35 of course.